action.yaml

```
@@ -10,6 +10,9 @@ inputs:
   minimum-approvals:
     description: Minimum number of approvals to progress workflow
     required: false
+  issue-title:
+    description: The custom subtitle for the issue.
```

The standard for this `action.yaml` is to not have a period at the end of the `description`.

```suggestion
    description: The custom subtitle for the issue
```